src/main/java/no/bibsys/lambda/deploy/handlers/DestroyHandler.java

```diff
@@ -0,0 +1,46 @@
+package no.bibsys.lambda.deploy.handlers;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import no.bibsys.cloudformation.Stage;
+import no.bibsys.lambda.deploy.handlers.templates.CodePipelineFunctionHandlerTemplate;
+import no.bibsys.lambda.deploy.requests.DeployEvent;
+import no.bibsys.lambda.responses.SimpleResponse;
+import no.bibsys.utils.Environment;
+import no.bibsys.utils.ResourceDestroyer;
+
+public class DestroyHandler extends CodePipelineFunctionHandlerTemplate<SimpleResponse> {
+
+    private final transient Environment environment;
+
+    public DestroyHandler(){
+        this(new Environment());
+    }
+
+
+    public DestroyHandler(Environment environment)  {
+        super();
+        this.environment=environment;
+
+    }
+
+    @Override
+    protected SimpleResponse processInput(DeployEvent input, String apiGatewayInputString,
+        Context context)
+        throws IOException, URISyntaxException {
+        Stage stage = Stage.currentStage();
```

Comment:
I'm thinking we should introduce some kind of task system and task interface when we want the init and destroyers to to more than one task

Reply:
This depends on each project. The only requirement should be that we can call the Init lambda function